src/components/views/dialogs/RoomSettingsDialog.js

```diff
@@ -64,7 +64,10 @@ export default class RoomSettingsDialog extends React.Component {
         const roomName = MatrixClientPeg.get().getRoom(this.props.roomId).name;
         return (
             <BaseDialog className='mx_RoomSettingsDialog' hasCancel={true}
-                        onFinished={this.props.onFinished} title={_t("Room Settings - %(roomName)s", {roomName})}>
+                        onFinished={this.props.onFinished}
+                        title={_t("Room Settings - %(roomName)s", {roomName})}
+                        fixedWidth={false}
```

Comment:
I guess it feels a bit strange to read `fixedWidth={false}` even though it is fixed (just via a different mechanism)... I assume we want to keep the "base 90vw, max 1000px" behaviour though, and maybe this is the easiest way...? Would your alternate proposals keep that?

Reply:
For example, `fixedWidth` would need to be more than just a single number it seems, since there's both a base size and max size, correct?

Reply:
Easiest way is probably just to make another PR so https://github.com/matrix-org/matrix-react-sdk/pull/2888 would be an alternative - I think I prefer it actually: it does avoid this particular confusion.